content/influxdb/cloud-dedicated/admin/users/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Manage users
+seotitle: Manage users and permissions in InfluxDB Cloud Dedicated
+description: >
+  Manage users and access to resources in your InfluxDB Cloud Dedicated cluster.
+  Assign user groups for role-based access control and security.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB Cloud
+weight: 101
+influxdb/cloud-dedicated/tags: [user groups]
+related:
+  - /influxdb/cloud-dedicated/reference/internals/security/
+  - /influxdb/cloud-dedicated/admin/tokens/ 
+---
+
+Manage users and access to resources in your {{% product-name %}} cluster.
+
+By assigning users to different groups based on the level of access they need,
+you can minimize unnecessary access and reduce the risk of inadvertent
+actions.
+User groups associate access privileges with user attributes--an important part of the
+Attribute-Based Access Control (ABAC) security model which grants access based on
+user attributes, resource types, and environment context.
+
+- [Available user groups](#available-user-groups)
+- [Manage users](#manage-users)
+- [Assign a user to a different group](#assign-a-user-to-a-different-group)
+- [Invite a user to your account](#invite-a-user-to-your-account)
+
+### Available user groups
+
+In {{% product-name %}}, users have "management" roles, such as creating and 
+deleting databases, viewing resource information, and provisioning
+[database tokens](/influxdb/cloud-dedicated/admin/tokens/database/) for reading and writing data. 
+
+A user can belong to the following groups, each with predefined privileges:
+
+<!-- Question: what are the "certain resources" below? -->
+
+- **Admin**: Read and write permissions on all resources.
+- **Member**: Read permission on certain resources and create permission for
+  database tokens; members can't delete or create databases or management tokens.
+- **Auditor**: Read permission on all resources; auditors can't modify resources.
+
+{{% note %}}
+#### Existing users are Admin by default
+
+With the release of user groups for {{% product-name %}}, all existing users
+in your account are initially assigned to the Admin group, retaining full
+access to resources in your cluster.
+{{% /note %}}
+
+### Manage users
+
+InfluxData uses Auth0 to create user accounts and assign users to groups
+in {{% product-name %}}.
```

Comment:
I think this section can be removed entirely, since Support handled user management right now and Auth0 is really just an implementation detail. User Groups are actually managed directly by Granite, not with Auth0.

Reply:
> I think this section can be removed entirely, since Support handled user management right now and Auth0 is really just an implementation detail. User Groups are actually managed directly by Granite, not with Auth0.

I assume the main point that we should keep is that we use use/integrate/interact(?) with OAuth or the account identity provider to create users. Any details or suggestions are welcome :)

Reply:
There are two cases where I think the fact that we use OAuth is relevant:

1. We only support OAuth for user authentication, so Clustered customers must configure a client/app within their identity provider that supports the necessary OAuth authentication flows.
2. In Cloud Dedicated, customers can work with Support to configure OAuth authentication, which _may_ require some OAuth-related information about their SSO provider.